app/src/main/java/io/github/droidkaigi/confsched/di/AppModule.java

```diff
@@ -87,7 +90,16 @@ public CompositeSubscription provideCompositeSubscription() {
     @Singleton
     @Provides
     public OrmaDatabase provideOrmaDatabase(Context context) {
-        return OrmaDatabase.builder(context).build();
+        return OrmaDatabase.builder(context)
+                .migrationStep(115, new ManualStepMigration.ChangeStep() {
+                    @Override
+                    public void change(@NonNull ManualStepMigration.Helper helper) {
+                        helper.renameColumn("Session", "placeId", "place");
+                        helper.renameColumn("Session", "speakerId", "speaker");
+                        helper.renameColumn("Session", "categoryId", "category");
```

Comment:
Just question for Orma.
If I want to change column name, I have to use this manual migration, right?